packages/temen/src/mapKeys/index.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * "object"와 동일한 값을 가진 객체를 만들고 "iterate"를 통해
+ * "object"의 각 열거형 문자열 키 속성을 이용해 새로운 생성된 키를 생성한합니다.
+ *
+ * 'object'의 키 속성을 'iterate'를 통해 문자열화합니다.
+ * iterate는 세 가지 인수(value, key, object)로 호출됩니다.
+ *
+ * @param {Object} object 반복할 객체
+ * @param {Function} iterate 반복당 호출 함수
+ * @returns {Object} 키 값이 새로 매핑 된 객체
+ *
+ * @example
+ * ```js
+ * mapKeys({ 'a': 1, 'b': 2 }, function(value, key) {
+ *   return key + value;
+ * });
+ * // => { 'a1': 1, 'b2': 2 }
+ * ```
+ */
+
+function mapKeys<T>(object: T, iterate: (value: T[keyof T], key: keyof T, object: T) => any) {
```

Comment:
```suggestion
function mapKeys<T>(object: T, iterate: (value: T[keyof T], key: keyof T, object: T) => string) {
```
그냥 string으로 강제해도 되지 않을까유??? 아니면 원래 오브젝트의 키 타입인 `string | number | symbol`이어도 될 것 같습니다.

Reply:
https://github.com/Lubycon/lubycon-frontend-libraries/pull/154/commits/a742db6e0d53cf1771f9f1bc3d8cf4fc6ced9fa2 심볼은 같이 주니 타입에러가..